tests/integration.rs

```diff
@@ -1124,35 +1164,38 @@ mod tests {
     }
 
     #[cfg_attr(not(feature = "mmio"), test)]
-    fn test_vhost_user_net() {
+    fn test_vhost_user_net(
```

Comment:
Oops! You need to remove `#[cfg_attr(not(feature = "mmio"), test)]` to let the tests compile correctly.